MapboxNavigationTests/MapboxNavigationTests.swift

```diff
@@ -276,13 +278,26 @@ class MapboxNavigationTests: XCTestCase {
         wait()
         assertImageSnapshot(matching: UIImageView(image: navigationMapView.snapshot()), as: .image(precision: 0.95))
         
+        navigationMapView.mapView.mapboxMap.setCamera(to: CameraOptions(center: centerCoordinate))
+        navigationMapView.show(routeResponse)
+        navigationMapView.showWaypoints(on: firstRoute)
+        wait()
+        assertImageSnapshot(matching: UIImageView(image: navigationMapView.snapshot()), as: .image(precision: 0.95))
```

Comment:
I'm just wondering if we also need to add the new expected snapshots [here](https://github.com/mapbox/mapbox-navigation-ios/tree/main/MapboxNavigationTests/__Snapshots__/iPhone14_5/15.5/MapboxNavigationTests)

I think it does not fail now because all such tests are [disabled](https://github.com/mapbox/mapbox-navigation-ios/pull/4071) on CI.

Reply:
I ran these locally and it succeeded. I thing it's kind of a point of new presentation methods to result in the same visuals as before, so reusing a snapshot is actually meant.